### PR TITLE
Rewrote test in modern way

### PR DESCRIPTION
**Pull request by AliIbadullayev**
* Rewrote test in modern way  according to #15
* Removed unused functions and unused modules from headers

### CODE REVIEW

File: src/course_bot/quiz.clj [hunk 1]
```diff
@@ -219,3 +219,8 @@
                    (do
                      (talk/send-text token id "Не понял, укажите корректный номер ответа (1, 2...).")
                      tx))))))
+
+(defn quiz-answ
+  "Function returns string with answers.
+  Parameter answers contains all answer of student."
+  [answers] (str "Quiz answers: " (str/join ", " answers)))
```
Comment (ryukzak) [inline src/course_bot/quiz.clj:226]: Sorry, but DRY principle in this case completely ruin readability of the tests and also, mix test and non-test code.

File: test/course_bot/general_test.clj [hunk 2]
```diff
@@ -70,90 +64,98 @@
...
-                        [0 "Restarted and notified: 1"]
-                        [1 "You can use /start once more."])
-
-      (start-talk "/start")
-      (ttalk/in-history *chat "Hi, I'm a bot for your course. I will help you with your work. What is your name (like in the registry)?"))))
-
-(talk/deftest restart-permitted-test [db *chat]
+        db (tt/test-database)
+        *chat (atom (list))
+        talk (tt/handlers  (general/start-talk db conf)
```
Comment (ryukzak) [inline test/course_bot/general_test.clj:71]: two spaces `handlers (general`

**Review by ryukzak — CHANGES_REQUESTED** [review]

File: src/course_bot/quiz.clj (hunk 1 above)
Comment (AliIbadullayev) [reply src/course_bot/quiz.clj]: deleted them

File: test/course_bot/general_test.clj (hunk 2 above)
Comment (AliIbadullayev) [reply test/course_bot/general_test.clj:71]: Must it be one space?

Comment (AliIbadullayev) [reply test/course_bot/general_test.clj]: fixed

**Comment by AliIbadullayev** [comment]
Conflicts are resolved

**Review by ryukzak — APPROVED** [review]

**Pull request merged**